deploy/olm-catalog/ibm-block-csi-operator-community/1.8.0/bundle-1.8.0.Dockerfile

```diff
@@ -9,5 +9,5 @@ LABEL operators.operatorframework.io.bundle.package.v1=ibm-block-csi-operator-co
 
 COPY manifests /manifests/
 COPY metadata /metadata/
-LABEL com.redhat.openshift.versions="v4.8-v4.9"
+LABEL com.redhat.openshift.versions="v4.7-v4.9"
```

Comment:
so for ocp 4.7, one can install 1.6.0 and 1.8.0, but not 1.7.0?

Reply:
I guess, yes